packages/common/src/tests/array.test.ts

```diff
@@ -0,0 +1,26 @@
+import { describe, it, expect } from 'vitest'
+import { arrayJoin, ARRAY_JOINER } from '../array.js'
+
+describe('arrayJoin', () => {
```

Comment:
probably better off using lodash from the utils pkg rather than rolling our own impl of arrayJoin

Reply:
Sure but I've not added the arrayJoin function in here. If you want to remove then that's good with me

Reply:
will do in followup ticket